### PR TITLE
allow analyzer version 0.35.x

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: pkgs/test/CHANGELOG.md [hunk 1]
```diff
@@ -9,11 +13,11 @@
 
 ## 1.5.0
 
-* Depend on `package:test_core` for core functionality. 
+* Depend on `package:test_core` for core functionality.
```
Comment (grouma) [inline pkgs/test/CHANGELOG.md]: Is this an intended change?

Comment (kevmoo) [reply pkgs/test/CHANGELOG.md]: Guessing this is @jakemac53 's markdown auto-format on save thingy

**Comment by kevmoo** [comment]
@jakemac53 – FYI I pushed a few things here to make it pass and ensure tests are run w/ dart_core + latest things

File: pkgs/test/CHANGELOG.md (hunk 1 above)
Comment (jakemac53) [reply pkgs/test/CHANGELOG.md]: Yep - figured I might as well leave it in since it cleans it up a bit

File: pkgs/test_core/pubspec.yaml [hunk 2]
```diff
@@ -43,3 +43,8 @@ dev_dependencies:
 dependency_overrides:
   test_api:
     path: ./../test_api
+
+  # to remove before publish
+  analyzer: ^0.35.0
+  test:
```
Comment (jakemac53) [inline pkgs/test_core/pubspec.yaml]: There are no tests in this package, and no package:test dependency... :P

Comment (kevmoo) [reply pkgs/test_core/pubspec.yaml]: I know...and yet if you don't add this, you won't get the latest version of pkg:test and few other things...it's weird

Comment (jakemac53) [reply pkgs/test_core/pubspec.yaml]: Ah it picks up a transitive dep on package:test.

The package:test version solve does get the latest analyzer though, and that is where the tests run, so I don't think this is necessary.

Comment (kevmoo) [reply pkgs/test_core/pubspec.yaml]: we'll nix all of the this before publish anyway 🤷‍♂️

Comment (jakemac53) [reply pkgs/test_core/pubspec.yaml]: Lol so this package has a bunch of dev deps, but no tests. As far as I can tell they can all be deleted which removes the circular dep.

**Comment by kevmoo** [comment]
SGTM!

**Pull request merged**